test/performance/alignment/edit_distance_benchmark_template.hpp

```diff
@@ -0,0 +1,99 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#pragma once
+
+#include <benchmark/benchmark.h>
+
+#include <seqan3/test/alignment/align_pairwise_edit_distance.hpp>
+#include <seqan3/test/performance/sequence_generator.hpp>
+#include <seqan3/test/performance/units.hpp>
+#include <seqan3/test/seqan2.hpp>
+
+// ----------------------------------------------------------------------------
+//  seqan3 edit distance pairwise alignment benchmarks
+// ----------------------------------------------------------------------------
+
+//!\brief This directly benchmarks the alignment algorithm without going through the whole align_pairwise function
```

Comment:
```suggestion
// This directly benchmarks the alignment algorithm without going through the whole align_pairwise function
```
I hope this doesn't appear in the documentation anyway but just for future copy and paste I would avoid it